src/core/companion.js

```diff
@@ -1,22 +1,28 @@
-import { TrackingEvents } from '../tracking/events'
+import { CompanionClicks } from '../clicks/companion'
+import { TrackingEvents } from './tracking-events'
 
 /**
  * Represents a single companion ad. Used in a {@link CompanionAds} creative.
  *
  * @class Companion
+ * @extends CompanionClicks
  * @author Tim De Pauw <tim.depauw@zentrick.com>
- * @copyright © 2016 Zentrick nv
+ * @copyright © 2018 Zentrick nv
  */
-export class Companion {
+export class Companion extends CompanionClicks {
   constructor () {
-    this._clickTrackings = []
+    super()
     this._trackingEvents = new TrackingEvents()
   }
 
+  // Attribute(s).
+
   /**
-   * The ID for this companion ad.
+   * The ID for this companion ad
+   *
+   * Bounded: 0-1
```

Comment:
"Bounded" only makes sense for array/list properties to me. In this case, it's an optional string, so the `@type` annotation already tells you that it's going to occur up to once.